README.md

```diff
@@ -146,6 +146,7 @@ wget -O .env https://raw.githubusercontent.com/immich-app/immich/main/docker/.en
 * Populate `UPLOAD_LOCATION` as prefered location for storing backup assets.
 * Populate a secret value for `JWT_SECRET`, you can use this command: `openssl rand -base64 128`
 * [Optional] Populate Mapbox value to use reverse geocoding.
+* [Optional] Populate `TZ` as your timezone, default is `Etc/UTC`. Immich uses your configured timezone to parse date in metadata (eg: `dateTimeOriginal`, `modifyDate`, ...) per photo. You can use `curl -s http://geoip.ubuntu.com/lookup | grep -oP '(?<=<TimeZone>)(.*)(?=</TimeZone>)'` or `curl -s http://ip-api.com/json/ | grep -oP '(?<=timezone":")(.*?)(?=")'` to fetch your current timezone.
```

Comment:
Probably better to have the user go to the link `http://geoip.ubuntu.com/lookup` and look for timezone, the curl command doesn't work on MacOs

Reply:
For macos, I will find another cmd.

Reply:
Probably better to make it clear that the TZ is for displaying the correct time zone on the web.

Reply:
@alextran1502 Can you help me correct this description by your word?